### ISSUE TITLE
Add Cuisine field to Supermarket preset

### ISSUE THREAD

**Issue by 1ec5** [openstreetmap/id-tagging-schema#623]
Many supermarkets specialize in one or more cuisines. The name suggestion index includes several full-service Asian and Latin American supermarket chains, for example. The Supermarket preset should have a Cuisine field just like the Restaurant preset. Needless to say, this doesn’t give mappers license to start tagging every supermarket with `cuisine=pizza` just because it sells frozen pizzas, but I think existing usage has remained pretty reasonable in that regard.

This would also apply to the Grocery preset proposed in #529.

**Comment by EvanCarroll** [comment]
Can you check this one @tyrasd. I believe the right solution here is [key:origin](https://wiki.openstreetmap.org/wiki/Key:origin).

> Tag to desribe the origin, e.g. the country of origin, for the goods of a shop.

This is how you'd mark up an Asian Grocery Store (or Arab, Turkish, etc etc).

Cuisine is for prepared food. While it's true that some supermarkets provide prepared food, most of them do this by way of a separate shop, inside a shop with different hours and contact information...

One such example of this is my local Central Market, you can see it here, the "Cafe" has different published hours. 

https://www.centralmarket.com/locations/houston

I would be cautious here because serving food isn't the primary function of grocery stores, and this only makes sense if,

* The contact information is the same
* The name is the same not Central Market Cafe, as above.
* The hours are same.

@1ec5 opinions about this?

**Comment by 1ec5** [comment]
I was unaware of the `origin` key; thanks for bringing it up. My intention was to encourage tagging of supermarkets based on the foods they sell overall, not specifically inside a particular department like the cafe. The cafe should be mapped as a separate node inside the supermarket, just like the bank branch, pharmacy counter, or florist, each of which typically keep disparate hours. `origin` is only half as common as `cuisine` on `shop=supermarket`, [810 to 415](https://overpass-turbo.eu/s/1n3r). It’s even more lopsided for `shop=grocery`, [97 to 8](https://overpass-turbo.eu/s/1n3u). I’m fairly confident that `cuisine` applies to the whole store in most of these cases, not some sub-feature within them. [Only 61 features](https://overpass-turbo.eu/s/1n3v) are dual-tagged as `shop=supermarket` `cuisine` _and_ `amenity`.

The two keys clearly overlap but aren’t semantically equivalent. A cuisine may or may not be an origin, especially given how `cuisine` has been polluted over the years by [non-cuisines](https://wiki.openstreetmap.org/wiki/Talk:Key:cuisine#Separate_cuisine_from_food_type), while an origin may or may not be for food. A Mexican supermarket would be known for selling food “from” Mexico but also sell other products from Mexico. On the other hand, there are food stores that specialize in non-regional cuisines like Jewish cuisine, and actually most of the products in an Asian grocery store in the U.S. actually come from the U.S. I guess it’s a question of how literally we interpret both keys.

This is tricky stuff! Previous efforts at reforming `cuisine` have raised more questions than answers, so personally I’d be inclined to cater to existing usage for now but would be open to moving things in a different direction. We should discuss the issue more broadly on the tagging mailing list or the Community Forum.

**Comment by EvanCarroll** [comment]
I agree, my reading here is _"cuisine_" is for prepared food. While shops/origin is for unprepared products you can buy. That's how the wiki currently documents it. Which makes sense for a client-engineering perspective. If I want to find "Turkish" cuisine I'm going to look for all cuisine=Turkish. If I want to buy Turkish goods I'd look for origin=Turkey.

This is pretty consistently documented in the wiki on both tag pages for key:origin ("origin of goods" for "shops which sell goods of foreign countries") and key:cuisine ("type of food served at an eating place") as well as key:shop (which only lists origin as useful combination).

However, the point you brought up about national origin is not documented at all on key:origin, that probably should be clarified. My own take on it is that for origin to be useful it must be about where they predominately consumed or originated, not where the national boundaries a specific product was produced.

For example,

* [Al Fakher](https://www.alfakher.com/en-us) is a famous brand of Egyptian hookahs and shisha. If they outsource to China does a store selling just Al Fakher markup with origin=China?
* For that matter, should Walmart be origin=China when it has 90%+ cohesion?

My assumption is that such an interpretation can't be useful to mappers or consumers.

**Comment by 1ec5** [comment]
Yeah, this will need some additional discussion. I thought it was more straightforward when suggesting this addition.

From your explanation, it sounds like we actually need both Cuisine and Origin fields for some shop presets. Aside from the documented distinction between prepared and non-prepared foods, there’s an inherent distinction that goes without saying: cuisines may be named after places but they aren’t places, so they can’t be origins. [This Mexican supermarket](https://www.openstreetmap.org/way/345802569) in California carries tomatoes, cabbage, etc. mainly because they’re used in Mexican cuisine, not strictly because tomatoes are grown in Mexico. On the other hand, [this Vietnamese fruit store](https://www.openstreetmap.org/node/4360926917) specializes in fruits grown in Vietnam, probably including some that aren’t really a part of Vietnamese cuisine. The two separate fields would allow mappers to indicate whatever is relevant about a given shop.

**Comment by 1ec5** [comment]
https://community.openstreetmap.org/t/how-to-tag-a-country-culture-specific-store/106151

**Comment by pandadeepimpact** [comment]
How about removing overlap between `cuisine` and `origin`? All countries and regions fall under `origin` (ex. American, Cantonese), leaving `cuisine` with just food types (ex. pizza, burger). Or maybe even chuck the term `cuisine` due to its ambiguity (the discussion seem to revolve on the different interpretations of the word "cuisine"), and replace it with something clearer (`food:option`?). I think it would be much more productive to remove ambiguities than trying to fix their definition, as it simplify things (as you wouldn't need sentences, or even worse, paragraphs, just to explain how to use it), and avoid the risk of errors from misunderstandings.

As for `origin`, I would agree with @EvanCarroll, it would be more of style than the source. Like for example, Korean dumplings often taste very different from Chinese ones, regarding of where they're made, or what country their ingredients came from.

As for the distinction between prepared foods, dry foods, frozen foods etc., I also think it would be better to introduce a separate tag for them, which would be clearer than implying them through the use of `cuisine` or `origin`, especially with the rising number of stores offering several of those at the same time.

**Comment by 1ec5** [comment]
[This forum thread](https://community.openstreetmap.org/t/how-to-tag-a-country-culture-specific-store/106151) would be the place to make these suggestions.

**Comment by matkoniecz** [comment]
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dsupermarket does not mention `cuisine=`, even as some kind of controversy

https://wiki.openstreetmap.org/wiki/Key:cuisine does not mention `supermarket=`

it sounds like it has not ended as accepted? Or has it been welcomed as a good idea and wiki documentation is missing?

**Comment by matkoniecz** [comment]
closing per above

`origin=` is documented and maybe making issue for it makes sense, maybe adding it using #2493 would be a good idea

**Comment by 1ec5** [comment]
As requested, I’ve [added documentation](https://wiki.openstreetmap.org/wiki/Special:Diff/3063051) about the de facto use of `cuisine=*` on `shop=supermarket` in the United States. I say _de facto_ because the rate of `cuisine=*` is apparently twice the proportion of actual ethnic supermarkets in the U.S., which means we have better coverage of ethnic supermarkets than mainstream ones. Go figure. Among these POIs, `cuisine=*` is used ten times as often as `origin=*`. The latter would be inaccurate for the vast majority of ethnic supermarkets anyways. None of the other alternatives suggested on the forum are documented anywhere in any context.

Would you be open to adding this field but only in the U.S. or other countries where this is an actual phenomenon?

**Comment by 1ec5** [comment]
One reason for the prevalence of `cuisine=*` on `shop=supermarket` is that [nine NSI entries](https://nsi.guide/index.html?t=*&k=shop&v=supermarket&tt=cuisine) specify `cuisine=*` (and none of them should specify `origin=*` as far as I can tell). All of them are exclusive to the U.S., except for H Mart, which also has locations in Canada and the United Kingdom.

**Comment by matkoniecz** [comment]
added NSI mention to docs and mentioned supermarkets at https://wiki.openstreetmap.org/wiki/Key:cuisine
